libs/ui-toolkit/src/components/link/link.tsx

```diff
@@ -27,5 +29,27 @@ export const Link = ({ className, children, ...props }: LinkProps) => {
     </a>
   );
 };
-
 Link.displayName = 'Link';
+
+export const ExternalLink = ({ children, className, ...props }: LinkProps) => (
```

Comment:
Suggestion (non-blocking): I'm not convinced this component is needed. I do like the use of the icon as a consistent feature of external links, but I'd rather see the Link component extended. Perhaps an optional boolean `external` prop that adds 'target="_blank"' and the icon?